core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/CloudbreakFlowInformation.java

```diff
@@ -73,7 +74,7 @@
 
     @Override
     public List<Class<? extends FlowConfiguration<?>>> getTerminationFlow() {
-        return Arrays.asList(StackTerminationFlowConfig.class, ClusterTerminationFlowConfig.class);
+        return Arrays.asList(StackTerminationFlowConfig.class, ClusterTerminationFlowConfig.class, ExternalDatabaseTerminationFlowConfig.class);
```

Comment:
Is this added to the parallel flows too?

Reply:
added now